railgun/test/integration/ingress_test.go

```diff
@@ -81,6 +82,63 @@ func TestMinimalIngress(t *testing.T) {
 		return false
 	}, ingressWait, waitTick)
 
+	t.Logf("removing the ingress.class annotation \"%s\" from ingress %s", ingressClass, ingress.Name)
+	ingress, err = cluster.Client().NetworkingV1().Ingresses("default").Get(ctx, ingress.Name, metav1.GetOptions{})
```

Comment:
nit: put the namespace name in a `const`

Reply:
I forgot about this you're right, changed to `corev1.NamespaceDefault` in 5b94b7e2a3d03e3957078703efa056967aef9640 :+1: